plugins/backstage-plugin-coder/src/components/CoderProvider/CoderAuthProvider.tsx

```diff
@@ -455,6 +449,10 @@ function FallbackAuthUi() {
   const fallbackRef = useRef<HTMLElement>(null);
   useLayoutEffect(() => {
     const fallback = fallbackRef.current;
+    const mainAppRoot = document.querySelector<HTMLElement>(
+      BACKSTAGE_APP_ROOT_ID,
+    );
```

Comment:
Just decided to tuck this in here to keep the layout effect logic more self-contained